okd-release-pipeline/buildconfigs/release-worker/tekton-worker-image.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
This file is checked in twice now, at `okd-release-pipeline/buildconfigs/release-worker/tekton-worker-image.yaml` and `buildconfigs/release-worker/tekton-worker-image.yaml`

Reply:
Well spotted. Removed `buildconfigs/release-worker/tekton-worker-image.yaml`

WDYT about the use of `batch-build` task in the `okd-release-pipeline`? It is referencing a task that is not copied to the `okd-release-pipeline/tasks` folder, but rather still under the `./tasks` folder directlly of this repo.
Let me know if you prefer I duplicate to ease the next step, taking `okd-release-pipeline` to another repo.